lib/opam/opam_make_opam.ml

```diff
@@ -0,0 +1,162 @@
+open Core.Std
+open Core_extended.Std
+open Async.Std
+
+let format_cmds
+  : String.t List.t -> String.t =
+  fun cmds ->  "[\n" ^
+               (List.fold ~init:""
+                  ~f:(fun acc el -> acc ^ "    [" ^ el ^ "]\n") cmds) ^
+               "]\n"
+
+let create_build_cmds
```

Comment:
I did refactor them. There still are functions named create_XXX_cmds but there is no duplication and they are there more for documentation.